Plotly ws/instructions update

euronion opened the pull request:
Hi there,

I updated the `environment.yml` file and streamlined the plotly workshop instructions a bit to only use a `conda` environment which automatically installs all dependencies. Hoping this makes entry for participants less familiar with Python/Anaconda/pip easier.

euronion committed:
Update conda environment file.

euronion committed:
Streamline setup instructions.

Bachibouzouk commented on plotly-dash/README.md:
````diff
@@ -1,62 +1,103 @@
+```
 
--you should be able to see it with the following command
+3. Move into the freshly cloned repositor
````
```suggestion
3. Move into the freshly cloned repository
```

Bachibouzouk commented on plotly-dash/environment.yml:
```diff
@@ -1,5 +1,9 @@
-- python=3.6
-- pip:
-   - -r requirements.txt
+  - pandas
```
Do you have an argument not to use a link to the `requirements.txt` file? My argument is that I prefer to have only one place where I define the dependencies

euronion replied:
I forgot to delete the `requirements.txt` from the branch.
The idea was to only use ` conda` with the `environment.yml` file and remove the `requirements.txt` alltogether.

Bachibouzouk replied:
The solution with `requirements.txt` linked in `environment.yml` works both for conda user and for non-conda users (like me), this is why I had implemented it. If `requirements.txt` is there it should not bother the beginners too much as they would only follow the instructions.

euronion replied:
Ah, gotcha. Shall I restore the original version with conda and pip?

Bachibouzouk replied:
If you don't mind yes, otherwise I can do it :)

euronion committed:
Update plotly-dash/README.md

Co-authored-by: Pierre Francois <pierre-francois.duc@netplus.ch>

euronion committed:
Reuse pips requirements.txt file for package dependencies instead of condas environment.yml.

euronion committed:
Make environment names consistent.

euronion committed:
Merge branch 'plotly-ws/instructions-update' of https://github.com/euronion/workshop into plotly-ws/instructions-update

euronion commented:
Done. I removed the `python=3.6` version shakle, added `pip` as explicit dependency for `conda` (otherwise it is annoying about `pip` not being an explicit dependency) and made the environment names coherent between the readme and the conda environment file. (I used your `workshop_env` name for the environment again).
